nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/storage/PutAzureDataLakeStorage.java

```diff
@@ -161,10 +161,10 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
                 if (dlsException.getStatusCode() == 409) {
                     if (conflictResolution.equals(IGNORE_RESOLUTION)) {
                         session.transfer(flowFile, REL_SUCCESS);
-                        String warningMessage = String.format("File with the same name already exists. " +
+                        final String logMessage = String.format("File with the same name already exists. " +
                                 "Remote file not modified. " +
                                 "Transferring {} to success due to %s being set to '%s'.", CONFLICT_RESOLUTION.getDisplayName(), conflictResolution);
-                        getLogger().warn(warningMessage, new Object[]{flowFile});
+                        getLogger().info(logMessage, flowFile);
```

Comment:
The approach should be adjusted to use `{}` placeholder arguments instead of `String.format()`.